git/git.go

```diff
@@ -122,7 +122,7 @@ func (c *gitConfig) Find(val string) string {
 
 // SetGlobal sets the git config value for the key in the global config
 func (c *gitConfig) SetGlobal(key, val string) {
-	simpleExec("git", "config", "--global", "--add", key, val)
+	simpleExec("git", "config", "--global", key, val)
 }
```

Comment:
The `--add` flag will not overwrite existing values. It just adds another entry with the same key.